.github/actions/setup-sentry/action.yml

```diff
@@ -158,54 +160,37 @@ runs:
         NEED_KAFKA: ${{ inputs.kafka }}
         NEED_SNUBA: ${{ inputs.snuba }}
         NEED_CLICKHOUSE: ${{ inputs.clickhouse }}
+        NEED_BIGTABLE: ${{ inputs.bigtable }}
         NEED_CHARTCUTERIE: ${{ inputs.chartcuterie }}
         WORKDIR: ${{ inputs.workdir }}
       run: |
         sentry init
 
         # redis, postgres are needed for almost every code path.
-        sentry devservices up redis postgres
+        services='redis postgres'
 
-        if [ "$BIGTABLE_EMULATOR_HOST" ]; then
-          sentry devservices up --skip-only-if bigtable
+        if [ "$NEED_CLICKHOUSE" = "true" ] || [ "$NEED_SNUBA" = "true" ]; then
+          services="${services} clickhouse"
         fi
 
-        # TODO: Use devservices kafka. See https://github.com/getsentry/sentry/pull/20986#issuecomment-704510570
```

Comment:
I fixed the issues mentioned in that PR, you need to use docker networking (like hostname `sentry_zookeeper` instead of `127.0.0.1`) for container to container comms. Local devenv is already running 6.2.0, so CI should as well. I'm aware production runs older versions, we should probably upgrade...